web/src/people.ts

```diff
@@ -475,8 +475,12 @@ export function get_recipients(user_ids_string: string): string {
         return my_full_name();
     }
 
-    const names = get_display_full_names(other_ids).sort();
-    return names.join(", ");
+    const names = get_display_full_names(other_ids);
+    const caseInsensitiveCompare = (a: string, b: string): number =>
+        a.toLowerCase().localeCompare(b.toLowerCase());
+    const sortedNames = names.sort(caseInsensitiveCompare);
```

Comment:
Please don't define a new comparison function; use the one we use elsewhere, in `util.ts` (see the comments about performance there as well).